core/block_svg.ts

```diff
@@ -659,11 +662,8 @@ export class BlockSvg
    * @internal
    */
   updateComponentLocations(blockOrigin: Coordinate) {
-    if (!this.rendered) {
-      // Rendering is required to lay out the blocks.
-      // This is probably an invisible block attached to a collapsed block.
-      return;
-    }
+    // TODO: make sure this doesn't have a performance impact
```

Comment:
do you have a bug for this, or already addressing in the follow up PR you mentioned?

Reply:
Thanks for the catch! I checked this but forgot to delete.